app/models/workflow.rb

```diff
@@ -5,7 +5,48 @@ class Workflow < ApplicationRecord
 
   validates :display_name, presence: true
 
-  def groups
-    transcriptions.group(:group_id).count
+  # return an array listing groups that belong to the workflow
+  # data contains following information per group:
+  #   * group id
+  #   * transcription count
+  #   * last updated at
+  #   * last updated by
+  def transcription_group_data
+    transcription_group_data = {}
+    group_transcriptions = ordered_transcription_groups.group_by { |group| group.group_id }
+    group_transcriptions.each do |group_id, data|
+      # take the first/latest update_at transcription
+      transcription = data.first
+      # get the count from the number of transcriptions in the group
+      # Note: this may be sub-optimal if a group can have a large number transcriptions
```

Comment:
Probably want to remove the editorializing in these two lines and open an issue it's a concern.

Reply:
Valid! I don't think it's a concern, since we do not expect to be working with large enough amounts of data to run into issues with slow queries. Not sure if it's worth it to make note of this as a potential issue in the future, in the case that the size of our data increases over time, since I wouldn't see this sort of thing as being classified as an actual current issue...